build/bakefiles/wx.bkl

```diff
@@ -47,7 +47,7 @@
         <!-- WXRC compiler is built by default: -->
         <!-- FIXME: this is dirty hack, better bakefile support for
              conditional and optional subprojects is needed -->
-        <action id="wxrc" cond="USE_XRC=='1'">
+        <action id="wxrc">
```

Comment:
I'm sorry, but why did `USE_XRC` disappear here? It should remain AFAICS...

Reply:
`USE_XRC` is disabled by the `configure` script if GUI support is disabled.
(in fact, the `--enable-xrc` parameter is not evaluated at all)

I tried patching `configure.in` to allow setting `USE_XRC` in non-GUI builds, but compilation failed... on GUI-related XRC classes.